charts/openshift-metering/templates/presto/presto-coordinator-statefulset.yaml

```diff
@@ -56,6 +56,11 @@ spec:
 {{ toYaml .Values.presto.spec.annotations | indent 8 }}
 {{- end }}
     spec:
+      hostAliases:
```

Comment:
Let's add some templating around the `useIPV6Networking` key:
```yaml
if {{- .Values.useIPV6Networking }}
...
{{- end }}
```
 
While I don't think adding a host alias for the ipv6 localhost will affect any ipv4-based installs, I still think we should separate single-stack ipv4 and ipv6 until we have a better idea of how to tackle dual-stack.